scripts/table_updates/tests/test_update_data_table.py

```diff
@@ -59,36 +133,35 @@ def config():
             "BLADDER": "syn26721150",
             "NSCLC2": "syn51318735",
             "CRC2": "syn52943208",
-            "RENAL": "syn59474241"
+            "RENAL": "syn59474241",
         },
-        "irr":{
+        "irr": {
             "BrCa": "syn24241519",
             "PANC": "syn25610271",
             "Prostate": "syn26275497",
             "BLADDER": "syn26721151",
             "NSCLC2": "syn51318736",
             "CRC2": "syn52943210",
-            "RENAL": "syn59474249"
+            "RENAL": "syn59474249",
         },
-        "tier1a_replacement_mapping":{
+        "tier1a_replacement_mapping": {
             "patient_characteristics_tier1a_replacement_mapping_table": "syn22222",
-            "cancer_panel_test_tier1a_replacement_mapping_table": "syn33333"
+            "cancer_panel_test_tier1a_replacement_mapping_table": "syn33333",
         },
-        "main_genie_release_version": "16.6-consortium",
+        "main_genie_release_version": "v1.0",
         "main_genie_data_release_files": "syn16804261",
         "main_genie_sample_mapping_table": "syn7434273",
         "patient_tier1a_column_list_to_be_replaced": [
             "naaccr_ethnicity_code",
             "naaccr_race_code_primary",
             "naaccr_race_code_secondary",
             "naaccr_race_code_tertiary",
-            "naaccr_sex_code"
+            "naaccr_sex_code",
         ],
-    "sample_tier1a_column_list_to_be_replaced": [
-        "cpt_sample_type", "cpt_seq_date"
-        ]
+        "sample_tier1a_column_list_to_be_replaced": ["cpt_sample_type", "cpt_seq_date"],
```

Comment:
The above is mainly about reformatting with only the main_genie_release_version being updated to a dummy version name.